optional/front-page.php

```diff
@@ -27,5 +27,5 @@
 		?>
 	</main><!-- #primary -->
 <?php
-get_sidebar();
+
```

Comment:
In Sidebars/component.php, its hard-coded to add "has-sidebar" selector to body class.

Reply:
No, the code in `Sidebars/component.php` does the opposite: it excludes the `has-sidebar` selector from this template. Unless I'm completely misreading the logic:

https://github.com/wprig/wprig/blob/v2.0/inc/Sidebars/Component.php#L90-L92

Reply:
You're right.